components/widgets/jenkins/index.js

```diff
@@ -0,0 +1,92 @@
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import styled from 'styled-components'
+import Widget from '../../widget'
+import Table, { Th, Td } from '../../table'
+import Badge from '../../badge'
+
+const JenkinsBadge = styled(Badge)`
+  background-color: ${props => {
+    switch (props.status) {
+      case 'FAILURE':
+        return props.theme.palette.errorColor
+      case 'UNSTABLE':
+        return props.theme.palette.warnColor
+      case 'SUCCESS':
+        return props.theme.palette.successColor
+      default:
+        return props.theme.palette.grey200
+    }
+  }}
+`
+
+export default class Jenkins extends Component {
+  static defaultProps = {
+    jobs: [],
+    title: 'Jenkins'
+  }
+
+  state = {
+    loading: false,
+    error: null
+  }
+
+  componentDidMount () {
+    this.loadInformation()
+  }
+
+  async loadInformation () {
+    this.setState({ loading: true, error: null })
+
+    const { jobs, url } = this.props
+
+    try {
+      const builds = await Promise.all(
+        jobs.map(async job => {
+          const res = await fetch(`${url}/job/${job.path}/lastBuild/api/json`)
+          const json = await res.json()
+
+          return {
+            name: job.label,
+            url: json.url,
+            result: json.result
+          }
+        })
+      )
+
+      this.setState({
+        loading: false,
+        builds
+      })
+    } catch (_) {
+      this.setState({
+        loading: false,
+        error: 'failed to load information'
+      })
+    }
+  }
+
+  render () {
+    const { loading, error, builds } = this.state
+    const { title } = this.props
+
+    return (
+      <Widget title={title} error={error} loading={loading}>
+        <Table>
+          <tbody>
+            {builds && builds.map(build => (
+              <tr key={`jenkins-${build.name}`}>
+                <Th>{build.name}</Th>
+                <Td>
+                  <a href={build.url} title={build.result}>
```

Comment:
maybe you will open it also on your local pc - there you could click and easily follow the broken build?
i can also remove it - thought it would be a good feature 👍